notebooks/official/custom/get_started_vertex_training_xgboost.ipynb

```diff
@@ -951,14 +898,14 @@
         "id": "cleanup:mbsdk"
       },
       "source": [
-        "# Cleaning up\n",
-        "\n",
+        "## Cleaning up\n",
         "To clean up all Google Cloud resources used in this project, you can [delete the Google Cloud\n",
         "project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#shutting_down_projects) you used for the tutorial.\n",
         "\n",
         "Otherwise, you can delete the individual resources you created in this tutorial:\n",
         "\n",
-        "- Custom Job (Custome Training job is remove in previous step)\n",
+        "- Custom Job (Custom Training job is removed in previous step)\n",
```

Comment:
add "the":
"Custom Job (Custom Training job is removed in the previous step)"

Reply:
added the change